### DISCUSSION TITLE
Looking to Hire a developer experienced in Serenity

### DISCUSSION THREAD

**Discussion by bkp-defense** [serenity-is/Serenity#6331] [Help]
Hello!  I work for a large Defense Company, based in US.  I am looking to hire a web developer who has experience using Serenity.  Let me know if you are interested.

**Comment by edwardch** [comment]
Hi @bkp-defense,

If you have additional info like links, roles, location, remuneration, etc that might help with getting some responses or referrals.

Cheers

**Comment by minhhungit** [comment]
@bkp-defense I want to propound myself :wink: , email it.minhhung@gmail.com

**Reply by mohesa78** [reply]
Interested , send details to mohesa@gmail.com
